papers/ice_nucleation_2024/perfect_calib.jl

```diff
@@ -17,29 +17,29 @@ for IN_mode in IN_mode_list
     y_truth = pseudo_data[1]
     coeff_true = pseudo_data[3]
 
-    output = calibrate_J_parameters(
+    output = calibrate_J_parameters_EKI(
         FT,
         IN_mode,
         params,
         IC,
-        y_truth,
+        y_truth[end-25:end],
```

Comment:
Might be good not to hard code 25 as the number but instead have some top level parameter that defines the window we average over at the end of the simulation. And then based on that compute the index value of the cutoff